examples/examples_nodejs_test.go

```diff
@@ -141,6 +142,21 @@ func TestNamingConventions(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAutoNamingOverlay(t *testing.T) {
+	var buf bytes.Buffer
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:           filepath.Join(getCwd(t), "autonaming-overlay"),
+			Stderr:        &buf,
+			ExpectFailure: true,
+			ExtraRuntimeValidation: func(t *testing.T, stackInfo integration.RuntimeValidationStackInfo) {
+				assert.Contains(t, buf.String(), "is too large to fix max length constraint of 64")
```

Comment:
Out of scope, but shouldn't the error message here be "... too large to **fit** max length constraint". Maybe it's a non-native speaker issue on my end.

I can cut a PR if we think this is wrong

Reply:
yeah I think it was just a typo